docs/reference/index.md

```diff
@@ -258,6 +258,55 @@ As a convenience for production deployments, with the below environment variable
 CS_DATABASE__INSTALL_AWS_RDS_CERT_BUNDLE="true"
 ```
 
+## Multitenant
```

Comment:
Should it be "Multitenancy" or perhaps something like "Multitenant mode"?

Reply:
Good call.